### ISSUE TITLE
Can't download "free with ads" YouTube movies

### ISSUE THREAD

**Issue by Whylickspittle** [Tyrrrz/YoutubeDownloader#363]
### Version

v1.10.1

### Platform

Windows 10

### Steps to reproduce

Video URL: https://www.youtube.com/watch?v=BJhIIOyUd9k

Download settings:
1080p (mp4)

Application settings:
Nothing changed from the initial state.

Steps:
Just download.

### Details

I tried to download the free videos in Youtube. So I switched my VPN to the US. But I found it seems that it didn't work for videos in the movie channel?
I could get the audio url and the video through request, and start  the download,but it's so slow.SO I Try to use the state-of-the-art youtube download tools for help!
actually  the 404 bug you have mentioned https://github.com/Tyrrrz/YoutubeDownloader/issues/33#issuecomment-514620495
,but i still seems need to be fixed.  Thank you! 

![image](https://github.com/Tyrrrz/YoutubeDownloader/assets/49609811/c49e1b4b-eac3-4292-9fb2-70145212d082)

### Checklist

- [X] I have looked through existing issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that that this bug is reproducible on the latest version of the application
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project

**Comment by Tyrrrz** [comment]
It should no longer crash but the downloads will still be slow on this kind of videos until https://github.com/Tyrrrz/YoutubeExplode/issues/706 is implemented (which is a lot of effort).

**Comment by Whylickspittle** [comment]
![image](https://github.com/Tyrrrz/YoutubeDownloader/assets/49609811/74fafd1b-c417-452e-872c-fc7502ee87f9)
link1:https://www.youtube.com/watch?v=6QcDE-eajjU     
error: need to transfer the itag '142'
link2:https://www.youtube.com/watch?v=6PV5qQ3Y7Vs
error: playback on other websites has been disadbled by the video owner


Hey,dude.Thank you for your help, due to my poor vpn, i couldn't express my gratitude in time.Now i met some more issues.
So I googled the YoutubeExplode,try to solve it.  By checking the  webpage source code  I found some new itags.  (in the next dialog)  
 I added some itags to make the YoutubeExplode could work.       Than i found the muxed streams didn't exist.
![image](https://github.com/Tyrrrz/YoutubeDownloader/assets/49609811/25d218e7-a354-4750-9f37-648ac09ae934)
![image](https://github.com/Tyrrrz/YoutubeDownloader/assets/49609811/4e29aa46-30a2-487d-a7da-048c3ade8b7b)


I found another issue that the "free with ad" movies   source-code page didn't have the feature like mp4a.40.2(  which   i think   it's the muxed download  source by observing other videos)
I'm a little confused.

**Comment by Whylickspittle** [comment]
146 =>1080,
      226 =>1080,
      227 =>1080,
      275 =>1080,
      360 =>1080,
      657 =>1080,
      658 =>1080,
      145 =>720,
      224 =>720,
      225 =>720,
      274 =>720,
      357 =>720,
      358 =>720,
      652 =>720,
      653 =>720,
      654 =>720,
      144 =>480,
      222 =>480,
      223 => 480,
      273 => 480,
      317 => 480,
      318 => 480,
      649 => 480,
      650 => 480,
      651 => 480,
      143 => 360,
      280 => 360,
      648 => 360,
      142 => 240,
      279 => 240,
      647 => 240,
      161 => 144,       
 the extra itag

**Comment by Whylickspittle** [comment]
I’m not sure that I should new a issue or just cliam my problems there.   I wish  the YoutubeExplode project could deal with the problems.   Additionally,I wish your hometown can be out of war ASAP！

**Comment by Tyrrrz** [comment]
As for muxed streams, yeah. The client YTE uses in order to fetch these videos does not return muxed streams. The other errors I'll take a look a bit later.

**Comment by Tyrrrz** [comment]
> link1:[youtube.com/watch?v=6QcDE-eajjU](https://www.youtube.com/watch?v=6QcDE-eajjU)
> error: need to transfer the itag '142'

I added the new itags, thanks.

> link2:[youtube.com/watch?v=6PV5qQ3Y7Vs](https://www.youtube.com/watch?v=6PV5qQ3Y7Vs)
> error: playback on other websites has been disadbled by the video owner

There is no workaround for this, unfortunately, at the moment. It's also blocked by https://github.com/Tyrrrz/YoutubeExplode/issues/706.

**Comment by Whylickspittle** [comment]
ok, I got it. It feels so great for me to give some useful feedback.  Thank you for your patient explanation
